packages/libnotify.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Libnotify < Package
+class Libnotify < Meson
   description 'A library for sending desktop notifications.'
   homepage 'https://git.gnome.org/browse/libnotify'
   version '0.8.1'
```

Comment:
Update to version [0.8.3](https://gitlab.gnome.org/GNOME/libnotify/-/tags/0.8.3?